src/Commands/ContinuousIntegration/TravisCi/Config/Config.php

```diff
@@ -0,0 +1,146 @@
+<?php
+
+namespace QuickStrap\Commands\ContinuousIntegration\TravisCi\Config;
+
+class Config extends AbstractOptions
+{
+    /** @var  string */
+    private $language;
+    
+    /** @var  string[] */
+    private $php = array();
```

Comment:
This property really represents the version entries of the matrix. Suggest renaming to versions.